openpecha/core/pecha.py

```diff
@@ -378,11 +382,22 @@ def reset_layers(self, base_name: str, exclude: List[LayerEnum] = []):
                 continue
             self.reset_layer(base_name, layer_name)
 
-    def publish(self):
+    def publish(self, asset_path:Path=None, asset_name:str=None):
+        asset_paths = []
         if not self.storage:
             self.storage = GithubStorage()
         self.storage.add_dir(path=self.pecha_path, description=self.about)
 
+        # Publishing assets in release
+        if asset_path:
+            repo_name = self.pecha_id
+            shutil.make_archive(asset_path.parent / asset_name, "zip", asset_path)
+            asset_paths.append(f"{asset_path.parent / asset_name}.zip")
+            create_release(
+                repo_name, prerelease=False, asset_paths=asset_paths, org=os.environ["OPENPECHA_DATA_GITHUB_ORG"], token=os.environ.get("GITHUB_TOKEN")
```

Comment:
```suggestion
                repo_name, prerelease=False, asset_paths=asset_paths, self.storage.org_name, token=self.storage.token)
```

use `self.storage` to get value for `org` and `token`